libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/NavigationMapboxMap.java

```diff
@@ -235,10 +235,12 @@ public void clearMarkers() {
    *
    * @param location to update the icon and query the map
    */
-  public void updateLocation(Location location) {
-    List<Location> locations = new ArrayList<>(1);
-    locations.add(location);
-    updateLocation(locations);
+  public void updateLocation(@Nullable Location location) {
```

Comment:
Should it be `@NonNull` 🤔?

```suggestion
  public void updateLocation(@NonNull Location location) {
```

Reply:
d'oh! that's what I get for being in a hurry.